### ISSUE TITLE
Pubsub Emulator support seems broken in 1.1.4

### ISSUE THREAD

**Issue by rdlowrey** [googleapis/nodejs-pubsub#794]
#### Environment details

  - OS: Ubuntu 18.04
  - Node.js version: 12.10
  - npm version: 6.10.3
  - `@google-cloud/pubsub` version: 1.1.4

#### Steps to reproduce

Works with `0.29.1`, doesn't work with anything after (including the just-released `1.1.4`).

```
const grpc = require('grpc');
const { PubSub } = require('@google-cloud/pubsub');

const client = new PubSub({
  projectId: 'my-test-dev',
  apiEndpoint: '127.0.0.1:8085', // <-- wherever pubsub is exposed
  grpc,
});

client.getTopics({ pageSize: 1 }).then(console.log);
```

Outputs:

```
(node:16706) UnhandledPromiseRejectionWarning: TypeError: Channel's second argument (credentials) must be a ChannelCredentials
    at ServiceClient.Client (/home/rdlowrey/dev/backend/node_modules/grpc/src/client.js:412:23)
    at new ServiceClient (/home/rdlowrey/dev/backend/node_modules/grpc/src/client.js:938:12)
    at GrpcClient.createStub (/home/rdlowrey/dev/backend/node_modules/google-gax/build/src/grpc.js:236:22)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:16706) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
```

We know you folks have been working hard trying to fix the recent issues. Using the javascript grpc lib doesn't work at all for us right now so we're trying to pass in the c++ `grpc` dependency directly.  After `0.29.1` it borks our local development setup. Changes seem to not be playing nice with the pubsub emulator.

We've tested running the above code by directly specifying the `apiEndpoint` constructor option and also with the `PUBSUB_EMULATOR_HOST` environment variable. Neither approach seems to work.

Thanks in advance.

**Comment by callmehiphop** [comment]
@rdlowrey thanks for reporting this issue! I'm currently investigating, but it appears that this only occurs when passing in the grpc C++ extension. For an immediate workaround I would suggest conditionally passing it in based on whether or not you're using the emulator.

**Comment by callmehiphop** [comment]
@rdlowrey just waiting for our bots to do their thing, but this fix for this will be released in `1.1.5` very shortly. Thanks for reporting!
